### ISSUE TITLE
--quiet but lot of outpu

### ISSUE THREAD

**Issue by romestylez** [hillu/local-spring-vuln-scanner#2]
When im using --quiet i still get "Access is denied." in the log. Would be great to just get info when a file is detected which is (possible) vulnerable

**Comment by doctore74** [comment]
I totally agree. I would like to integrate the testtool into checkmk as a checkplugin.
